We've disabled math plugin at the moment.
This patch is probably not needed when -DRUNNER_MATH=ON
--- src/dialog.cpp	2021-04-11 09:01:40.000000000 +0000
+++ src.mod/dialog.cpp	2021-05-20 16:11:59.594667000 +0000
@@ -498,8 +498,7 @@
     if (res)
     {
         hide();
-        if (!qobject_cast<const MathItem*>(command)) // don't clear math results
-            ui->commandEd->clear();
+        ui->commandEd->clear();
     }
 
 }
